README.md

```diff
@@ -1113,6 +1113,37 @@ In either case:
        an attribute when `self` is an ActiveRecord model: `self.guest = user`.
     3. Referencing the current instance's class: `self.class`.
 
+* <a name="freeze-constants"></a>When defining arrays and hashes meant to be
+    constants, make sure to call `freeze` on them. ([More on this][ruby-freeze].)
+    <sup>[[link](#freeze-constants)]</sup>
+
+    The reason is that Ruby constants are actually mutable. Calling `freeze`
+    ensures they are not mutated and are therefore truly constant and 
+    attempting to modify them will raise an exception. 
+
+    ```ruby
+    class COLOR
+      # Each color
+      RED = 0
```

Comment:
I was first going to write it that way after reading the link explaining the efficiency gain, but then I decided to scope down the rule to just arrays and hashes because we totally don't do this in our code base at all. 

Totally open to expanding this rule to strings, and in fact any type really (even RED = 0.freeze is ideal because we don't want some other code to re-assign RED), but again none of our code does it, so it felt a bit hypocritical. 

Thoughts?